CHANGELOG.md

```diff
@@ -1,33 +1,32 @@
 # Release History
 
 ## 1.1.0-beta.2 (Unreleased)
```

Comment:
You will need to rev the date in here before releasing.  Up to you if you want to split the PRs and do this one line change tomorrow

```suggestion
## 1.1.0-beta.2 (2020-11-10)
```